lib/Twig/Loader/Filesystem.php

```diff
@@ -19,9 +19,9 @@ class Twig_Loader_Filesystem implements Twig_LoaderInterface
     /** Identifier of the main namespace. */
     const MAIN_NAMESPACE = '__main__';
 
-    protected $paths = array();
-    protected $cache = array();
-    protected $errorCache = array();
+    private $paths = array();
+    private $cache = array();
+    private $errorCache = array();
```

Comment:
@fabpot the question is why the Symfony bundle notation is implemented as a child of the Twig_Loader_Filesystem (I guess the reason is because Twig_Loader_Chain was not available at this time and was added only later when wanting to support Twig_Loader_Filesystem in Silex alongside the Twig_Loader_Array).
We could implement the loading of ``Bundle:Folder:template`` as a separate standalone loader and rely on the loader chain instead (this could be considered as a BC break in Symfony though, and so be harder to apply in 2.3+).